openpype/hosts/maya/plugins/publish/validate_frame_range.py

```diff
@@ -27,6 +27,7 @@ class ValidateFrameRange(pyblish.api.InstancePlugin):
                 "yeticache"]
     optional = True
     actions = [openpype.api.RepairAction]
+    exclude_families = ["model"]
```

Comment:
If we add this to settings, we'll solve the production situation right now and can focus on doing it properly as Roy suggests in v4